source-academy/frontend
# Update editor tab file paths when files & directories are renamed

### Description

Similar issue as #2386. Part of #2176.

### Type of change


### How to test

The Folder mode is not yet ready for production and is disabled programmatically. Enable Folder mode for testing by changing the condition to `false`:
https://github.com/source-academy/frontend/blob/b90a7a249be4969e6127e08eedc06900db2a78d5/src/pages/playground/Playground.tsx#L609-L622

Check that renaming files or directories containing files that are open as editor tabs also results in the corresponding editor tabs being renamed.
